setup.md

```diff
@@ -2,44 +2,81 @@
 
-__These lessons are designed for running on a UNIX based system (either linux or OSX), they have not been tested on Windows.__
-
 ### VSCode and Benten installation
 
-Download and install VSCode binaries from [https://code.visualstudio.com/download](https://code.visualstudio.com/download)
+Download and install [VSCode](https://code.visualstudio.com/).
 
 Open VSCode and search for [Benten](https://marketplace.visualstudio.com/items?itemName=sbg-rabix.benten-cwl) in the marketplace. The name of the client extension is Rabix/benten. Follow the usual method to install the extension.
 
 The VSCode Benten extension will require the Benten server to be installed too. It will prompt you to do this the first time you activate the extension.
 
-
 ### CWL installation
 
-Python3 and the CWL reference runner are required for this lesson. We recommend that you use [Anaconda](https://www.anaconda.com/distribution/) for installing these.
-Once you have installed Anaconda you can create a virtual environment which includes the necessary libraries using the command:
+To install the cwltool on Linux or Windows, follow these [instructions](https://github.com/common-workflow-language/cwltool#install).
+
+
+## Files
+
+You will need to install some example files for this lesson. In this tutorial we will use RNA sequencing data.
+
+### Setting up a practice repository
+For this tutorial some existing tools are needed to build the workflow. These existing tools will be imported via GitHub. 
+First we need to create an empty git repository for all our files. To do this, use this command:
 ~~~
-$ conda create -n cwl -c conda-forge cwl_runner
+$ git init novice-tutorial-exercises
 ~~~
 {: .language-bash}
 
-That virtual environment can be activated after creation using the command (within the terminal of VSCode):
+Next, we need to import bio-cwl-tools with this command:
 ~~~
-$ conda activate cwl
+$ git submodule add https://github.com/common-workflow-library/bio-cwl-tools.git
 ~~~
 {: .language-bash}
-After this the required tools should be available.
 
+### Downloading sample and reference data
+Move inside the `novice-tutorial-exercises` directory and download the data:
+~~~
+$ mkdir rnaseq
+$ cd rnaseq
+$ wget --mirror --no-parent --no-host --cut-dirs=1 https://download.jutro.arvadosapi.com/c=9178fe1b80a08a422dbe02adfd439764+925/
+~~~
+{: .language-bash}
+
+### Downloading or generating STAR index
+To run the STAR tool index files generated from the reference files are needed.
+This is a large download (4 GB), so it is also possible to generate these files yourself.
+
+#### Downloading
+~~~
+$ mkdir hg19-chr1-STAR-index
+$ cd hg19-chr1-STAR-index
+$ wget --mirror --no-parent --no-host --cut-dirs=1 https://download.jutro.arvadosapi.com/c=02a12ce9e2707610991bd29d38796b57+2912/
+~~~
+{: .language-bash}
 
+#### Generating 
+Create `chr1-star-index.yaml` in the the `novice-tutorial-exercises` directory:
```

Comment:
Could the writing of this configuration file for creating the index files be folded into the lessons? Or, instead, provided in the git template repository?

Reply:
It could, yes →https://github.com/carpentries-incubator/cwl-novice-tutorial/issues/52#issuecomment-975645452